README.md

```diff
@@ -4,38 +4,85 @@ Dummy Credentials repository to demonstrate how to bootstrap a tower instance wi
 
 ## Getting Started
 
-This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project. As this project contains sensitive information, it should only be used locally.
+This project serves as the inventory source for the [Ansible Tower Configuration](https://github.com/integr8ly/ansible-tower-configuration) project.
 
-## Usage
+## Usage 
 
-The example below runs the `bootstrap.yml` playbook using this project as the inventory source, where `--ask-vault-pass` is the password used to access the encrypted variables.
+The projects `SAMPLE_CREDENTIAL_CONFIG.yml` file contains a list of variables which need to be changed to suit your own environment. Once these variables have been changed, the `bootstrap.yml` playbook is used to consume this configuration file,encrypt any variables containing sensitive information using the supplied Ansible vault password, and then place each of the variables into the relevant group_vars file.
 
-```bash
-ansible-playbook -i <path-to-tower-dummy-credentials-project> playbooks/bootstrap.yml -e tower_host=<tower-host> -e tower_username=<tower-username> -e tower_password=<tower-password> -e tower_environment=<tower-environment> --ask-vault-pass
-```
+This project should be forked and made private before proceeding with the following setup steps.
 
-## Ansible-Vault
+## Setup
 
-Credentials stored in this project are encrypted using [Ansible Vault](https://docs.ansible.com/ansible/latest/user_guide/vault.html). Ansible vault supports the encryption of entire [files](https://docs.ansible.com/ansible/latest/user_guide/vault.html#creating-encrypted-files), and also individual [string values](https://docs.ansible.com/ansible/latest/user_guide/vault.html#use-encrypt-string-to-create-encrypted-variables-to-embed-in-yaml) that can embedded within YAML files. The default encrypted string value of all variables within this project along with the password to decrypt these variables is `CHANGEME`.
+1. Clone this project
 
-### Files
+     * 
+         ```bash
+         cd <local_projects_directory>
+         ```
+     *
+         ```bash
+         git clone https://github.com/integr8ly/tower_dummy_credentials
+         ```
 
-Configuration variables required for the Openshift-Ansible installation process are contained within the `provisioning_vars.yml.j2` file. This file is encrypted using Ansible Vault and can be decrypted using the command below.
+2. Copy and rename the `SAMPLE_CREDENTIAL_CONFIG.yml` file locally. As the local copy of the config will sensitive information in plaintext, it should only be stored and referenced locally, and caution taken to not check it into a public repository.
 
 ```bash
-ansible-vault view provisioning_vars.yml.j2
+cp SAMPLE_CREDENTIAL_CONFIG.yml <path-on-local-machine>/local_credentials_config.yml
```

Comment:
Could remove the path on local machine section and just set this to: 
```
cp SAMPLE_CREDENTIAL_CONFIG.yml local_credentials_config.yml
```
That way users can just copy/paste the command directly